integration/querydsl/testsuite/pom.xml

```diff
@@ -349,6 +349,9 @@
                                 <!-- Careful, we need this otherwise the ordering will be wrong... -->
                                 <user.country>US</user.country>
                                 <user.language>en</user.language>
+                                <!-- Dumb, dumber, Oracle -->
```

Comment:
You also need `<jdbc.url>jdbc:oracle:thin:@localhost:1521:XE</jdbc.url>` and `<jdbc.password>Oracle18</jdbc.password>`